### ISSUE TITLE
Display document type badges

### ISSUE THREAD

**Issue by jmartinm** [inspirehep/inspire-next#689]
Now that we store the document type inside the record metadata and show it in facets:

![screen shot 2016-01-07 at 11 22 50 am](https://cloud.githubusercontent.com/assets/580727/12168215/16f56cf0-b531-11e5-88f4-65eb6a0798ea.png)

Users have shown interest in having for each search result a 'badge' that shows the document type so that at first glance it would be easier to find what you are looking for.

![screen shot 2016-01-07 at 11 25 21 am](https://cloud.githubusercontent.com/assets/580727/12168294/7ffd9f7e-b531-11e5-851f-272cdfd9a3eb.png)

One possible location of such badge would be on the right side of a search result (on top of the year) although alternatives are welcome.

**Comment by bing13** [comment]
Worth trying and testing, especially since we seem to have a principled spot for it.  

More useful data is great, unless it starts overloading the interface.

**Comment by eamonnmag** [comment]
In a previous mockup, I had this included.

![screen shot 2016-01-08 at 15 31 04](https://cloud.githubusercontent.com/assets/282396/12200054/ddfe2a44-b61c-11e5-94de-92031ab58db7.png)

**Comment by annetteholtkamp** [comment]
In my view this option is nicer than moving it far to the right.

- Annette

**Comment by jmartinm** [comment]
> In my view this option is nicer than moving it far to the right.

Yes, the only thing is to have a consistent position for it. Apart from the title and authors the rest of the information appears or not depending on the record so we just need to make sure that such badges have a constant position to not distract too much.

**Comment by jmartinm** [comment]
I still like this idea, so if there are resources would be nice to try it and show it to some users for feedback with and without the badge.
